src/CoreMedia/CMSync.cs

```diff
@@ -549,25 +626,61 @@ public CMClock CopyUltimateMasterClock ()
 		}
 
 #if !NET
-		[iOS (9,0)][Mac (10,11)]
+		[iOS (9,0)][Mac (10,11), NoMacCatalyst]
+		[Deprecated (PlatformName.iOS, 9, 0, message: "Use 'CMTimebaseGetMasterTimebase' instead.")]
+		[Deprecated (PlatformName.TvOS, 9, 0, message: "Use 'CMTimebaseGetMasterTimebase' instead.")] 
+		[Deprecated (PlatformName.MacOSX, 10, 11, message: "Use 'CMTimebaseGetMasterTimebase' instead.")]
+		[Deprecated (PlatformName.WatchOS, 6, 0, message:  "Use 'CMTimebaseGetMasterTimebase' instead.")]
+#else
+		[UnsupportedOSPlatform ("ios15.0")]
+		[UnsupportedOSPlatform ("tvos15.0")]
+		[UnsupportedOSPlatform ("maccatalyst15.0")]
```

Comment:
```suggestion
		[UnsupportedOSPlatform ("maccatalyst")]
```
Does this make a difference